src/sunmatrix/sparse/sunmatrix_sparse.c

```diff
@@ -299,31 +290,34 @@ SUNMatrix SUNSparseFromBandMatrix(SUNMatrix Ad, sunrealtype droptol,
 /* ----------------------------------------------------------------------------
  * Function to create a new CSR matrix from a CSC matrix.
  */
-int SUNSparseMatrix_ToCSR(const SUNMatrix A, SUNMatrix* Bout)
+SUNErrCode SUNSparseMatrix_ToCSR(const SUNMatrix A, SUNMatrix* Bout)
 {
-  if (A == NULL) { return (SUNMAT_ILL_INPUT); }
-  if (SM_SPARSETYPE_S(A) != CSC_MAT) { return (SUNMAT_ILL_INPUT); }
+  SUNFunctionBegin(A->sunctx);
+  SUNAssert(SUNMatGetID(A) == SUNMATRIX_SPARSE, SUN_ERR_ARG_WRONGTYPE);
+  SUNAssert(SM_SPARSETYPE_S(A) == CSC_MAT, SUN_ERR_ARG_OUTOFRANGE);
```

Comment:
Like above, should be `SUN_ERR_ARG_INCOMPATIBLE` or `SUN_ERR_ARG_WRONGTYPE` instead?

Reply:
Actually, should this be an error or should we just copy `A` to `B`?